### PR TITLE
test: Add repr test for #2763

### PR DESCRIPTION
**Pull request by breathx**

### CODE REVIEW

File: pallets/gear/src/tests.rs [hunk 1]
```diff
@@ -701,6 +701,46 @@ fn reply_deposit_gstd_async() {
...
+                RuntimeOrigin::signed(USER_1),
+                constructor,
+                calls.encode(),
+                BlockGasLimitOf::<Test>::get(),
+                0,
+            ));
+            let mid = get_last_message_id();
+            run_to_next_block(None);
+
+            mid
```
Comment (shamilsan) [inline pallets/gear/src/tests.rs:725]: `mid` is too short for me, I'd use `msg_id`.

File: pallets/gear/src/tests.rs [hunk 2]
```diff
@@ -701,6 +701,46 @@ fn reply_deposit_gstd_async() {
...
+        };
+
+        // message wakes some message id and waits
+        let waited_mid = execute(Calls::builder().wake([0u8; 32]).wait());
```
Comment (shamilsan) [inline pallets/gear/src/tests.rs:729]: And so on.
```suggestion
        let waited_msg_id = execute(Calls::builder().wake([0u8; 32]).wait());
```

**Review by shamilsan — APPROVED** [review]
Great 👍

**Review by grishasobol — APPROVED** [review]

File: pallets/gear/src/tests.rs (hunk 1 above)
Comment (breathx) [reply pallets/gear/src/tests.rs:725]: We use such abbreviation across the project, together with pid (program_id). Furthermore, it literally "expanded" in function name - `*message_id`

Comment (breathx) [reply pallets/gear/src/tests.rs]: Nvm, changed to `*msg_id`

**Pull request merged**